package-lock.json

```diff
@@ -5338,12 +5338,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Hmm, so strangely enough, on the mac, after running `npm install`, the `package-lock.json` file will be modified to **remove** this `optional` tag. 😒 

Do you think you could investigate a solution to this?

Can you maybe check the existing NPM issues in the NPM repo for this problem to see if anyone else is also running into this and any possible solutions?

If there are no solutions then we'll just have to be OK with the file constantly being modified in PRs, but that's kinda annoying.